multiqc/modules/percolator/percolator.py

```diff
@@ -0,0 +1,229 @@
+        "charge3",
+        "m0",
+        "COMET:lnExpect",
+        "enzInt",
+        "isotope_error",
+        "dm",
+    ],
+    "ms2pip": [
+        "ionb_min_abs_diff",
+        "ionb_mse",
+        "iony_std_abs_diff_norm",
+        "ionb_mse_norm",
+        "ionb_std_abs_diff_norm",
+        "dotprod",
+        "cos_norm",
+        "iony_std_abs_diff",
+        "iony_mean_abs_diff_norm",
+        "dotprod_norm",
+        "abs_diff_Q2_norm",
+        "iony_spearman",
+        "ionb_min_abs_diff_norm",
+        "iony_max_abs_diff_norm",
+        "dotprod_iony_norm",
+        "ionb_abs_diff_Q2",
+        "max_abs_diff",
+        "dotprod_ionb_norm",
+        "abs_diff_Q3_norm",
+        "iony_min_abs_diff_norm",
+        "cos_iony_norm",
+        "spec_pearson",
+        "std_abs_diff",
+        "min_abs_diff_norm",
+        "ionb_max_abs_diff",
+        "dotprod_iony",
+        "ionb_mean_abs_diff_norm",
+        "max_abs_diff_iontype",
+        "mean_abs_diff",
+        "iony_max_abs_diff",
+        "cos_iony",
+        "ionb_std_abs_diff",
+        "iony_pearson",
+        "ionb_max_abs_diff_norm",
+        "mean_abs_diff_norm",
+        "abs_diff_Q1",
+        "ionb_pearson_norm",
+        "ionb_abs_diff_Q1_norm",
+        "min_abs_diff",
+        "ionb_mean_abs_diff",
+        "spec_spearman",
+        "iony_mean_abs_diff",
+        "abs_diff_Q1_norm",
+        "iony_min_abs_diff",
+        "ionb_spearman",
+        "iony_abs_diff_Q3_norm",
+        "min_abs_diff_iontype",
+        "cos_ionb_norm",
+        "abs_diff_Q2",
+        "abs_diff_Q3",
+        "spec_pearson_norm",
+        "ionb_abs_diff_Q2_norm",
+        "std_abs_diff_norm",
+        "dotprod_ionb",
+        "cos_ionb",
+        "ionb_abs_diff_Q3",
+        "iony_mse_norm",
+        "iony_pearson_norm",
+        "spec_mse_norm",
+        "ionb_abs_diff_Q3_norm",
+        "iony_abs_diff_Q1_norm",
+        "ionb_pearson",
+        "max_abs_diff_norm",
+        "spec_mse",
+        "iony_mse",
+        "ionb_abs_diff_Q1",
+        "iony_abs_diff_Q2_norm",
+        "iony_abs_diff_Q1",
+        "iony_abs_diff_Q3",
+        "cos",
+        "iony_abs_diff_Q2",
+    ],
+    "deeplc": [
+        "rt_diff",
+        "predicted_retention_time",
+        "predicted_retention_time_best",
+        "observed_retention_time_best",
+        "rt_diff_best",
+        "observed_retention_time",
+    ],
+}
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    The module parses results generated by [Percolator](https://github.com/percolator/percolator),
+    a semi-supervised learning framework for peptide identification from shotgun proteomics datasets.
+    """
+
+    def __init__(self):
+        super(MultiqcModule, self).__init__(
+            name="Percolator",
+            anchor="percolator",
+            href="https://github.com/percolator/percolator",
+            info="Semi-supervised learning for peptide identification from shotgun proteomics datasets.",
+        )
```

Comment:
Please add doi (set it to empty list if it's unpublshed), see:

https://docs.seqera.io/multiqc/development/modules#multiqcmodule-class